docs/cell_census_schema_0.0.1.md

```diff
@@ -0,0 +1,853 @@
+
+All datasets included in the Cell Census MUST be of [CELLxGENE dataset schema version 3.0.X](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md). The following data constraints are imposed on top of the CELLxGENE dataset schema.
+
+#### Species
+
+Only observations (cells) with an  [`organism_ontology_term_id`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#organism_ontology_term_id) value of either “NCBITaxon:10090” for *Mus musculus* or “NCBITaxon:9606” for *Homo sapiens* MUST be included.
+
+Only features (genes) with a [`feature_reference`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#feature_reference) value of either “NCBITaxon:10090” for *Mus musculus* or “NCBITaxon:9606” for *Homo sapiens* MUST be included
+
+#### Multi-species data constraints
+
+Per the CELLxGENE dataset schema, [multi-species datasets MAY contain observations (cells) of a given organism and features (genes) of a different one](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#general-requirements), as defined in [`organism_ontology_term_id`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#organism_ontology_term_id) and [`feature_reference`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#feature_reference) respectively. 
+
+For a given multi-species dataset, the table below shows all possible combinations of organisms for both observations and features. For each combination, inclusion criteria for the Cell Census is provided.
+
+<table>
+<thead>
+  <tr>
+    <th>Observations (cells) from</th>
+    <th>Features (genes) from</th>
+    <th>Inclusion criteria</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>"NCBITaxon:9606" for <i>Homo sapiens</i> AND "NCBITaxon:10090" for <i>Mus musculus</i></td>
+    <td>"NCBITaxon:9606" for Homo sapiens</td>
+    <td>Only observations from "NCBITaxon:9606" for <i>Homo sapiens</i> MUST be included</td>
+  </tr>
+  <tr>
+    <td>"NCBITaxon:9606" for <i>Homo sapiens</i> AND "NCBITaxon:10090" for <i>Mus musculus</i></td>
+    <td>"NCBITaxon:10090" for <i>Mus musculus</i></td>
+    <td>Only observations from "NCBITaxon:10090" for <i>Mus musculus</i> MUST be included</td>
+  </tr>
+  <tr>
+    <td>"NCBITaxon:9606" for <i>Homo sapiens</i> AND "NCBITaxon:10090" for <i>Mus musculus</i></td>
+    <td>"NCBITaxon:9606" for <i>Homo sapiens</i> AND "NCBITaxon:10090" for <i>Mus musculus</i></td>
+    <td>All observations MUST NOT be included</td>
+  </tr>
+</tbody>
+</table>
+
+#### Assays
+
+Assays are defined in the CELLxGENE dataset schema in [`assay_ontology_term_id`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#assay_ontology_term_id). Only data from the following assays that sample RNA molecules from individual cells MUST be included in the Cell Census:
```

Comment:
same confusion as above - as I read it, other assays MAY be included (define what MUST be excluded)